# patch to enable linux-x86 arch in GAMESS DDI socket compile
# script

--- gamess/ddi/compddi	2005-07-23 20:32:02.000000000 +0000
+++ gamess-gentoo/ddi/compddi	2005-09-05 12:20:02.000000000 +0000
@@ -14,7 +14,7 @@
 #      fuji-pp32,fuji-pp64,hpux32,hpux64,ibm32,ibm64,ibm64-sp,ibm-bg,
 #      linux-ia64,linux-pc,mac32,macG5,necsx,sgi32,sgi64,sun32,sun64
 #
-       set TARGET = ibm64
+       set TARGET = linux-pc
 
 #  2. Choose DDI communication layer, the only legal values are
 #                   sockets,mpi,shmem,lapi
@@ -470,11 +470,11 @@
 #       select only one rather than two trailing underscores below.
   if($TARGET == linux-pc) then
 
-     set CC = 'gcc'
+     set gentoo-CC = 'gcc'
      set CFLAGS = "-DLINUX -O3 -fstrict-aliasing -I./include"
      set CLIBS  = "-lpthread"
 
-     set F77_OPTS = '-DINT_SIZE=int -D_UNDERSCORES=2'
+     set gentoo-F77_OPTS = '-DINT_SIZE=int -D_UNDERSCORES=2'
 
      set AR_FLAGS     = 'cr'
      set RANLIB_FLAGS = ' '
